src/force/CabanaPD_ForceModels_PMB.hpp

```diff
@@ -121,6 +128,56 @@ struct ForceModel<LinearPMB, Fracture> : public ForceModel<PMB, Fracture>
     using base_type::s0;
 };
 
+template <typename TemperatureType>
+struct ForceModel<PMB, Elastic, TemperatureDependent, TemperatureType>
+    : public ForceModel<PMB, Elastic, TemperatureIndependent>,
+      BaseTemperatureModel<TemperatureType>
+{
+    using base_type = ForceModel<PMB, Elastic, TemperatureIndependent>;
+    using base_temperature_type = BaseTemperatureModel<TemperatureType>;
+    using base_model = PMB;
+    using fracture_type = Elastic;
+    using thermal_type = TemperatureDependent;
+
+    using base_type::c;
+    using base_type::delta;
+    using base_type::K;
+
+    // Thermal coefficients
```

Comment:
temp0 is not a coefficient. Perhaps say "// Thermal parameters"?

Reply:
Fixed